angular.json

```diff
@@ -39,7 +39,16 @@
                 "output": "doc"
               }
             ],
-            "styles": ["src/styles.scss"],
+            "styles": [
```

Comment:
@hupf Für die Anforderung, dass das Popup via CSS positioniert werden kann, habe ich ein eigenes Bundle gemacht (die Styles der Componente sind ansonsten inline, wohl weil die Componente in der eigentlichen App nicht enthalten ist). Ist das ok so oder gibt es da bessere Lösungen für Angular?

Reply:
@spahrson Das wüsste ich vom Schiff aus auch nicht gerade...

Ohne, dass ich es jetzt genauer angeschaut habe: hast du mal mit `npm run analyze` geschaut wie das Bundle aussieht? Wird da im separaten Bundle das ganze Bootstrap nochmal reingezogen?

Reply:
@hupf nein das sieht soweit gut aus. Bootstrap bleibt nur im styles.scss.